badger/cmd/pick_table_bench.go

```diff
@@ -0,0 +1,225 @@
+/*
+ * Copyright 2021 Dgraph Labs, Inc. and Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cmd
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"runtime/pprof"
+	"sort"
+	"testing"
+
+	"github.com/dgraph-io/badger/v3"
```

Comment:
fix import order

Reply:
how so?  `go fmt` didn't make any changes

Reply:
It is just a good convention to separate dgraph and non-dgraph imports. First non-dgraph imports and then dgraph imports. Look at any other file.

Reply:
@joshua-goldstein the order needs to be opposite. First non-dgraph imports and then dgraph imports

Reply:
done